js/components/utils.js

```diff
@@ -953,6 +953,27 @@ var Utils = {
     }
     return false;
   },
+
+  /**
+   * the function returns the maximum width of the elements
```

Comment:
I posted earlier that we should look into doing this with CSS, but in case this needed, we should add a more detailed description. It's not clear what this maximum width is or what it is used for.

Reply:
\+